### DISCUSSION TITLE
--qos default value is 128 made me think I didn't have to write a value. Would not start!

### DISCUSSION THREAD

**Discussion by 38github** [jamulussoftware/jamulus#1965] [Documentation]
Since the guide on the web site says that --qos defaults to 128 I thought that I had to not write a value after the command but this caused the server (FreeBSD) to not start. I had to enter a value for the server to work. Maybe the Command Line Options page should be made clearer?

**Comment by dcorson-ticino-com** [comment]
Default is the value if the command line option is not used.
This is like all the other command line options (and in all programs that I know), if you use the option in the command line it uses that value you give it, if you do not use the command line option it uses the default.

**Reply by 38github** [reply]
Thank you for the explaination. Maybe it should be made clear that it is always used unless set as value 0?

**Reply by ghost** [reply]
Actually; 'it is always used even when set as value 0' because 0 is a valid number here.

The only way for Jamulus to not start because of --qos is from a syntax error typed at the command line (or your script). The error is:

Jamulus: '--qos' needs a numeric argument between '0' and '255'.

which includes '0' and '255'.
